### PR TITLE
dont wrap toolbar icons in mui array template fix #1985

### LINKED ISSUE #1985: Material-UI array icons wrap in some cases
When rendering the array template (material-ui theme), there are cases when the icons (up/down etc) will wrap to new line.
In the [Example](https://codesandbox.io/s/react-jsonschema-form-array-bug-w94ff) shrink the browser and at some point the icons will wrap.

I have a quick fix idea, in `DefaultArrayItem` when rendering the item `Grid` [here](https://github.com/rjsf-team/react-jsonschema-form/blob/v2.3.0/packages/material-ui/src/ArrayFieldTemplate/ArrayFieldTemplate.tsx#L80) add `style={{overflow: 'auto'}}`, this will prevent the first cell to take the entire space.

(version: 2.3.0)

### PR DESCRIPTION
**Pull request by ValYouW**
Fix #1985

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [X ] **I'm adding or updating code**
  - [X ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

**Comment by ValYouW** [comment]
@epicfaace If you get the time, can you please take a look? thx.

**Comment by ValYouW** [comment]
@stanlemon @epicfaace Just stumbled in my code at the workaround I have due to #1994 which reminded me this PR. I updated it to latest version. Looks ready to me...

**Comment by stanlemon** [comment]
Changes look good to me! 👍🏻

File: packages/material-ui/src/IconButton/IconButton.tsx [hunk 1]
```diff
@@ -8,21 +8,23 @@ import Remove from "@material-ui/icons/Remove";
...
 
 type IconButtonProps = MuiIconButtonProps & {
   icon: string;
+  iconProps?: object;
 };
 
 const IconButton = (props: IconButtonProps) => {
-  const { icon, className, ...otherProps } = props;
+  const { icon, className, iconProps, ...otherProps } = props;
+  var IconComp = mappings[icon];
```
Comment (epicfaace) [inline packages/material-ui/src/IconButton/IconButton.tsx:24]:
```suggestion
  const IconComp = mappings[icon];
```

**Review by epicfaace — APPROVED** [review]
Yeah, this looks good. Thanks!

**Comment by ValYouW** [comment]
Thx!

**Pull request merged**